frontend/www/js/omegaup/components/arena/RunDetails.vue

```diff
@@ -49,42 +43,74 @@
             >
-              <td></td>
-              <td class="text-center">{{ problem_case.name }}</td>
-              <td class="text-center">{{ problem_case.verdict }}</td>
-              <template v-if="data.show_diff !== 'none'">
-                <td class="text-center">
-                  {{ showDataCase(data.cases, problem_case.name, 'in') }}
+              <tr>
+                <td></td>
+                <td class="text-center">{{ problem_case.name }}</td>
+                <td class="text-center">{{ problem_case.verdict }}</td>
+                <td class="score">
+                  {{
+                    problem_case.contest_score
+                      ? problem_case.contest_score
+                      : problem_case.score
+                  }}
                 </td>
-                <td class="text-center">
-                  {{ showDataCase(data.cases, problem_case.name, 'out') }}
+                <td class="center" width="10">
+                  {{ problem_case.max_score ? '/' : '' }}
                 </td>
-                <td class="text-center">
-                  {{ showDataCase(data.cases, problem_case.name, 'output') }}
+                <td>
+                  {{ problem_case.max_score ? problem_case.max_score : '' }}
                 </td>
-                <td
-                  class="text-center diff"
-                  v-html="showDiff(data.cases, problem_case.name)"
-                ></td>
+              </tr>
+              <template v-if="data.show_diff !== 'none'">
+                <tr>
+                  <td colspan="6">{{ T.wordsInput }}</td>
+                </tr>
+                <tr>
+                  <td colspan="6">
+                    <pre>{{
+                      showDataCase(data.cases, problem_case.name, 'in')
+                    }}</pre>
+                  </td>
+                </tr>
+                <tr>
+                  <td colspan="6">{{ T.wordsOutputExpected }}</td>
+                </tr>
+                <tr>
+                  <td colspan="6">
+                    <pre>{{
+                      showDataCase(data.cases, problem_case.name, 'out')
+                    }}</pre>
+                  </td>
+                </tr>
+                <tr>
+                  <td colspan="6">{{ T.wordsOutputObtained }}</td>
+                </tr>
+                <tr>
+                  <td colspan="6">
+                    <pre>{{
+                      showDataCase(
+                        data.cases,
+                        problem_case.name,
+                        'contestantOutput',
+                      )
+                    }}</pre>
+                  </td>
+                </tr>
+                <tr>
+                  <td colspan="6">{{ T.wordsDifference }}</td>
+                </tr>
+                <tr>
+                  <td
+                    colspan="6"
+                    v-html="showDiff(data.cases, problem_case.name)"
+                  ></td>
```

Comment:
```suggestion
                  <td colspan="6">
                    <pre
                      v-html="showDiff(data.cases, problem_case.name)"
                    ></pre
                  ></td>
```

si no, no se van a desplegar bien las líneas.

Reply:
Listo